h2o-hive/src/main/java/water/hive/JdbcHiveMetadata.java

```diff
@@ -197,10 +210,9 @@ private StorableMetadata readStorableMetadata(Map<String, Object> tableInfo) {
         List<Column> partitionKeys
     ) throws SQLException {
         if (partitionKeys.isEmpty()) {
-            return Collections.emptyList();
+            return emptyList();
```

Comment:
Empty and unmodifiable at the same time would serve us best ?
```suggestion
            return unmodifiableList(emptyList());
```

Reply:
emptyList is unmodifiable